Makefile

```diff
@@ -106,15 +107,20 @@ build-container:
 $(MANPAGES): %: %.md
 	@sed -e 's/\((skopeo.*\.md)\)//' -e 's/\[\(skopeo.*\)\]/\1/' $<  | $(GOMD2MAN) -in /dev/stdin -out $@
 
-docs: $(MANPAGES)
+.PHONY: image.docs
+image.docs:
+	@git clone https://github.com/containers/image.git image.docs 2>/dev/null || true
+	@(cd image.docs; git checkout ${IMAGEVERSION} -b ${IMAGEVERSION}; git filter-branch --prune-empty --subdirectory-filter docs -- --all 2>/dev/null;for i in *.md; do $(GOMD2MAN) -in $$i -out $${i//.md}; done) || true
+
+docs: $(MANPAGES) image.docs
 
 docs-in-container:
 	${CONTAINER_RUNTIME} build ${BUILD_ARGS} -f Dockerfile.build -t skopeobuildimage .
 	${CONTAINER_RUNTIME} run --rm --security-opt label=disable -v $$(pwd):/src/github.com/containers/skopeo \
 		skopeobuildimage make docs $(if $(DEBUG),DEBUG=$(DEBUG)) BUILDTAGS='$(BUILDTAGS)'
 
 clean:
-	rm -f skopeo docs/*.1
+	rm -rf image.docs
```

Comment:
Why does this remove the pre-existing cleanups?

Reply:
Bug, fixed